bindings/python/vara-feature/pybind_FeatureModelBuilder.cpp

```diff
@@ -0,0 +1,94 @@
+#include "pybind11/attr.h"
+#include "pybind_common.h"
+#include "vara/Feature/FeatureModel.h"
+#include "vara/Feature/FeatureModelBuilder.h"
+#include "vara/Feature/FeatureModelTransaction.h"
+
+#include "pybind11/detail/common.h"
+#include "pybind11/pybind11.h"
+#include <pybind11/stl.h>
+#include <pybind11/complex.h>
+#include <pybind11/functional.h>
+#include <pybind11/chrono.h>
+#include <filesystem>
+#include <iostream>
+#include <bits/stdc++.h>
+
+namespace vf = vara::feature;
+namespace py = pybind11;
+
+void init_feature_model_builder_module(py::module &M){
+    py::class_<vf::FeatureModelBuilder>(M,  "FeatureModelBuilder")
+        .def(py::init([](){
+                      return vf::FeatureModelBuilder();
+                      }))
+        .def(
+            "make_binary_feature",
+            [](vf::FeatureModelBuilder &FMB, std::string feature_name, bool Opt){
+                return FMB.makeFeature<vf::BinaryFeature>(feature_name); 
+            }    
+        )   
```

Comment:
```suggestion
        .def(
            "make_binary_feature",
            [](vf::FeatureModelBuilder &FMB, std::string feature_name, bool Opt){
                return FMB.makeFeature<vf::BinaryFeature>(feature_name); 
            }
        )
```

Please don't add any trailing whitespaces.

Reply:
Also, please add a bit of documentation to the bindings.
For example:
```
        .def(
            "make_binary_feature",
            [](vf::FeatureModelBuilder &FMB, std::string feature_name, bool Opt){
                return FMB.makeFeature<vf::BinaryFeature>(feature_name); 
            },
            R"pbdoc(Create a new binary `Feature` with the given `feature_name`.)pbdoc"
        )
```